config/local-setup/dns-provider/endpoint/kustomization.yaml

```
@@ -0,0 +1,13 @@
+nameSuffix: -endpoint
+
+generatorOptions:
+  disableNameSuffixHash: true
+  labels:
+    app.kubernetes.io/part-of: dns-operator
+    app.kubernetes.io/managed-by: kustomize
+    kuadrant.io/endpoint_gvr: "kuadrant.io_v1alpha1.dnsrecords"
+    kuadrant.io/zone_record_label: "kuadrant_zone_record"
+
+secretGenerator:
+  - name: dns-provider-credentials
+    type: "kuadrant.io/endpoint"
```

```suggestion
generatorOptions:
  disableNameSuffixHash: true
  labels:
    app.kubernetes.io/part-of: dns-operator
    app.kubernetes.io/managed-by: kustomize

secretGenerator:
  - name: dns-provider-credentials
    type: "kuadrant.io/endpoint"
    literals:
      - ENDPOINT_GVR=kuadrant.io_v1alpha1.dnsrecords
      - ENDPOINT_ZONE_RECORD_LABEL=kuadrant.io/endpoint-provider-zone-record=true
```

As per the design I'd expect these to be setting configuration in the secret